blueprint/core/blueprint_EcmascriptEngine.cpp

```diff
@@ -273,34 +150,38 @@ namespace blueprint
     //==============================================================================
     juce::var EcmascriptEngine::invoke (const juce::String& name, const std::vector<juce::var>& vargs)
     {
-        // Evaluate the target string on the context, leaving the result on the stack
-        // TODO: Return specific error code if name not in stack?
-        if (!detail::safeEvalString(ctx, name, errorHandler))
-            return juce::var::undefined();
-
-        // Ensure requested function exists on the stack
-        duk_require_function(ctx, -1);
+        try {
```

Comment:
Should we indent brace here as per juce style guide?

i.e. 

```
try
{
 // stuff
}
catch (const auto &e)
{
// stuff
}
```

Reply:
Yea, I should really add a clang-format/clang-tidy step :)

Reply:
Heh. Funnily enough we've just been looking at the same thing!